globus_cli/config.py

```diff
@@ -86,12 +100,30 @@ def get_output_format():
     return lookup_option(OUTPUT_FORMAT_OPTNAME)
 
 
-def get_auth_refresh_token():
-    return lookup_option(AUTH_RT_OPTNAME)
+def get_auth_tokens():
+    return {
+        'refresh_token': lookup_option(AUTH_RT_OPTNAME),
+        'access_token': lookup_option(AUTH_AT_OPTNAME),
+        'access_token_expires': int(lookup_option(AUTH_AT_EXPIRES_OPTNAME))
```

Comment:
Because `lookup_option` can return `None`, this check fails if you kill your own config file or remove the `access_token_expires` field.
Applies to Transfer as well, of course.

To confirm a fix, move your config file and try `globus auth get-identities 'abc@globusid.org' --debug`

I've run into similar issues before -- should `lookup_option` be changed to take a callable named `type`?